src/Tickets/Emails/Email_Handler.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * Tickets Emails Handler.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails;
+
+/**
+ * Class Email_Handler.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Email_Handler extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * Event Tickets Emails post type.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	const POSTTYPE = 'tec_tickets_emails';
+
+	/**
+	 * Registered emails.
+	 *
+	 * @since TBD
+	 *
+	 * @var Email_Abstract[]
+	 */
+	protected $emails = [];
+
+	/**
+	 * Emails
+	 *
+	 * @since TBD
+	 *
+	 * @var string[]
+	 */
+	protected $default_emails = [
+		\TEC\Tickets\Emails\Email\Ticket::class,
+	];
+
+	/**
+	 * Binds and sets up implementations.
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+		foreach ( $this->default_emails as $email_class ) {
```

Comment:
We should use the `get_all` method here maybe. Otherwise the classes would only load the default ones here, instead of any externally registered. I might be wrong tho,